MobileCenter/MobileCenterTests/MSChannelDefaultTests.m

```diff
@@ -79,7 +79,9 @@ - (void)testEnqueuingItemsWillIncreaseCounter {
 
   // When
   for (int i = 1; i <= itemsToAdd; i++) {
-    [self.sut enqueueItem:[MSAbstractLog new] withCompletion:nil];
+    id logMock = OCMPartialMock([MSAbstractLog new]);
+    OCMStub([logMock isValid]).andReturn(YES);
```

Comment:
Can it be better to put these lines in a separate function? They are repeated 8 times